libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -114,6 +114,9 @@ class AP_InertialSensor : AP_AccelCal_Client
     const Vector3f     &get_accel(uint8_t i) const { return _accel[i]; }
     const Vector3f     &get_accel(void) const { return get_accel(_primary_accel); }
 
+    // return the number of accel/gyro instances.  Assume n_accel = n_gyro.
+    uint8_t get_count(void) const { return _accel_count; }
```

Comment:
we already have get_accel_count() and get_gyro_count()
A get_count() is just misleading, as we can have a different gyro count to accel count